src/MerchantCenter/AccountService.php

```diff
@@ -0,0 +1,491 @@
+<?php
```

Comment:
💅 I'd add the prefix In the file names.

`src/MerchantCenter/MerchantCenterAccountService.php`
`src/Ads/AdsAccountService.php`

There are inconsistencies on this over the codebase. For example, sometimes we use the prefix, others do not. 

Maybe out scope this PR but I think the code will be cleaner with the prefixes and consistency over the names.

Reply:
I don't recall us discussing this previously, maybe it's something we can bring up for discussion. My opinion differs though. I personally don't like really long names, and feel that the whole reason we are using namespaces is so we don't have to create really unique class names.

```
Automattic\WooCommerce\GoogleListingsAndAds\MerchantCenter\AccountService
```

In my opinion that looks cleaner and we don't need to repeat the word MerchantCenter twice, otherwise how far do we go with a class that might be multiple levels down.